docs/source/instructions_lassen.rst

```diff
@@ -0,0 +1,116 @@
+Instructions for running DLIO Benchmark on Lassen, LLC.
+=============
+
+'''''''''''''''''''''''
+Installation
+'''''''''''''''''''''''
+On Lassen on the login node: 
+
+* **Clone the github repository**:
+
+.. code-block:: bash
+
+	git clone https://github.com/argonne-lcf/dlio_benchmark
+	cd dlio_benchmark/
+
+* **Use conda**:
+
+.. code-block:: bash
+
+	# Setup the required channels:
+	conda config --prepend channels https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/
+
+	# Create and activate environment
+	conda env create --prefix ./dlio_env_ppc --file environment-ppc.yaml --force
+	conda activate ./dlio_env_ppc
+
+	#Install other dependencies and make sure it finishes successfully with no errors:
+	python -m pip install .
+
+
+**NOTE**: if problem with mpi4py make sure that mpi is pointing to the right version of gcc.
+Do not install packages using the $conda install command but rather install all required versions of packages.
+To check versions of mpicc and gcc:
+
+.. code-block:: bash
+
+	gcc --version
+	mpicc --version
+
+To specify a new link for gcc:
+
+.. code-block:: bash
+
+	which mpicc
+	export CC='which mpicc'
+	export CXX=mpic++
+
+'''''''''''''''''''''''
+Generate synthetic data that DLIO will use
+'''''''''''''''''''''''
+
+**On Lassen generate data with the use of JSRUN scheduler**:
+* Arguments to use:
+1) --bind packed:4 (to bind tasks with 4 CPUs)
+2) --smpiargs="-gpu" (enables gpu support)
+3) --nrs x (allocation of x node, it can be set to to 1, 2, 4 etc On Lassen we have 756 compute nodes)
+4) --rs_per_host 1 (resources per node)
+5) --tasks_per_rs y (y processes per resourse set/per node, it can be set to to 1, 2, 4 etc On Lassen we have 44 processes per node)
```

Comment:
It can be set to 1,2, or 4 as Lassen has 4 GPUs per node.

Reply:
For the tasks per rs, right? The nrs could be more

Reply:
Yes

Reply:
Fixed